MovieGPT-api/src/main/java/lt/techin/group/project/exception/GenreNotFoundException.java

```diff
@@ -5,7 +5,7 @@
 
 @ResponseStatus(HttpStatus.NOT_FOUND)
```

Comment:
for duplicated data we can use HttpStatus.CONFLICT as a response, for more accurate status.

Reply:
We can also add:
 @ExceptionHandler(GenreAlreadyExistsException.class)
    public ResponseEntity<ErrorResponse> handleGenreAlreadyExistsException(GenreAlreadyExistsException ex) {
        ErrorResponse errorResponse = new ErrorResponse(
                HttpStatus.CONFLICT.value(),
                "CONFLICT",
                ex.getMessage()
        );
        return ResponseEntity.status(HttpStatus.CONFLICT).body(errorResponse);
    }
    
   to GlobalExceptionHandler to catch this specific exception

Reply:
Fixed, pushed new code